contracts/settling_game/L02_Resources.cairo

```diff
@@ -116,44 +124,53 @@ func claim_resources{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_che
     # check days greater than zero
     assert_not_zero(days)
 
+    # get wonder tax percentage 
+    let ( wonder_tax ) = IL04_Calculator.calculate_wonder_tax(contract_address=calculator_address)
+    let wonder_tax_rel_perc = (80 * wonder_tax) / 100
```

Comment:
Sorry, I didn't catch this last time.

Normal division (using `/`) in Cairo is a [problem](https://www.cairo-lang.org/docs/hello_cairo/intro.html#the-primitive-type-field-element-felt). You should use `unsigned_div_rem` from the `starkware.cairo.common.math` module, probably everywhere where there's a div op.